public_html/assets/css/styles.css

```diff
@@ -483,20 +479,20 @@ body.work-page .nav-wrap hr.title-underline {display:none;}
 /* -------------------------------------------------------------------------------------------------------------
     MEDIA STYLES
 -------------------------------------------------------------------------------------------------------------*/
-@media screen and (min-width:960px) {
-    section#welcome .logogroup {display:inline-block; margin:0 10px;}
-}
-
 @media screen and (min-width:768px) {
     section#client-compliments .container12:last-of-type .column6 {margin-bottom:0;}
 }
 
 @media screen and (max-width:1160px) {
+    section#welcome .logogroup {font-size:14px;}
```

Comment:
at slightly smaller screens, the logo group shrinks slightly using font-size changes. this helps it fit a tad better in the shrinking white space